backend/psql/query.ts

```diff
@@ -1 +1,9 @@
-export const QUERY_GET_ALL_DAOS = "SELECT * FROM sigma_daos";
+export const QUERY_GET_DAOS_COUNT = "SELECT COUNT(*) FROM sigma_daos";
+
+/**
+ * Paginated query to fetch only the required information from database instead of all
+ * @param pageNumber Current page number to fetch data for.
+ * @param pageSize Number of data to show in single page/fetch.
+ */
+export const getPaginatedQuery = (pageNumber: number, pageSize: number) =>
```

Comment:
the name is too general, we are DAOs. Let's rename this function.

Reply:
ok. converted to `getPageQuery`

Reply:
you should rename to something with DAO in name, `getPaginatedDaos` for example.